test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/lang/StringTest.scala

```diff
@@ -279,9 +283,9 @@ class StringTest {
     assertEquals(s.length - 1, s.offsetByCodePoints(s.length - 1, 0))
     assertEquals(s.length, s.offsetByCodePoints(s.length, 0))
 
-    assertThrows(classOf[IndexOutOfBoundsException], s.offsetByCodePoints(-3, 4))
-    assertThrows(classOf[IndexOutOfBoundsException], s.offsetByCodePoints(6, 18))
-    assertThrows(classOf[IndexOutOfBoundsException], s.offsetByCodePoints(30, 2))
+    assertThrows(classOf[IndexOutOfBoundsException], s.offsetByCodePoints(-3, -4))
+    assertThrows(classOf[IndexOutOfBoundsException], s.offsetByCodePoints(6, -18))
+    assertThrows(classOf[IndexOutOfBoundsException], s.offsetByCodePoints(30, -2))
```

Comment:
Why are the negative signs necessary (IIUC they don't harm / change the test, but it feels the change is unnecessary)?

Reply:
Oops, I meant to change the ones in `offsetByCodePointsBackward`.